### ISSUE TITLE
[F-R] Use Device Brightness When Disable Player Swipe Gestures

### ISSUE THREAD

**Issue by RickyM7** [Quickdesh/Animiru#23]
### Describe your suggested feature

Maybe a option to choose your favorite behavior should be nice.

### Other details

I think this should be the default behavior when disabling gestures.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.4.4](https://github.com/Quickdesh/Animiru/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by RickyM7** [comment]
I jut realised that is on the app already, I'm really sorry!

**Comment by quickdesh** [comment]
Option has been added in player settings

**Comment by RickyM7** [comment]
The behavior of this feature is a bit strange, as the ideal would be to change the brightness of the phone and the player at the same time, but at the moment it is necessary to exit and return to playback for the brightness to be updated.

Animiru version: v0.14.1.4

**Comment by quickdesh** [comment]
Could you take a video and upload it here? I am a little confused as to what you mean

**Comment by RickyM7** [comment]
> Could you take a video and upload it here? I am a little confused as to what you mean

Unfortunately I can't record it, but you can reproduce it on your device with these steps:
 1. Play an episode of an anime
 2. Increase/decrease brightness via phone bar
 3. You will see that the playback brightness has not changed, so exit and enter playback and you will see that the brightness has now changed.
 Note: The option to remember the last brightness must be disabled, of course.

**Comment by warior456** [comment]
This is also in the main app, for me my brightness bar is still locked when those options are toggled

**Comment by killerlearner** [comment]
Yeah it isn't using system brightness when brightness gesture is disabled and also last brightness gesture
